Inputting/InputDefinitions/TouchscreenDragInputDefinition.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Inputting.InputDefinitions
+{
+    /// <summary>
+    /// A touchscreen drag input is an input in the form of <c>x1,y1>x2,y2</c>, e.g. <c>120,215>80,170</c>,
+    /// with <c>x</c> and <c>y</c> being within 0 (inclusive) and the specified max width/height (exclusive).
+    /// The resulting input's effective text will be "drag".
+    /// The dragged coordinates will be passed via additional data in the form of a 4-tuple <c>(x1, y1, x2, y2)</c>.
+    /// </summary>
+    public struct TouchscreenDragInputDefinition : IInputDefinition
+    {
+        public const string EffectiveText = "drag";
```

Comment:
Maybe I'm silly but shouldn't the EffectiveText be the normalized version of the input instead of "drag"?